cocos/physics/bullet/instantiated.ts

```diff
@@ -148,7 +148,7 @@ export function waitForAmmoInstantiation (): Promise<void> {
         { default: bulletWasmUrl },
         { default: bulletAsmFactory },
     ]) => {
-        if (shouldUseWasmModule()) {
+        if (shouldUseWasmModule() && bulletWasmUrl) {
```

Comment:
Why should check the url now?

Reply:
If we build `asmjs` only, but the browser supports `wasm`, `shouldUseWasmModule()` will return true and go to the the logic of initializing wasm which is not what we want, it should go to `initializing asmjs` logic. 
Therefore, we need to check `bulletWasmUrl`  which is empty when we only use asmjs .